gax-java/gax/src/main/java/com/google/api/gax/rpc/EndpointContext.java

```diff
@@ -100,6 +104,11 @@ public static EndpointContext getDefaultInstance() {
   @Nullable
   public abstract String transportChannelProviderEndpoint();
 
+  public abstract boolean useS2A();
```

Comment:
This can be package private now.
```suggestion
  abstract boolean useS2A();
```

Reply:
Done in https://github.com/googleapis/sdk-platform-java/pull/3326/commits/03eb991dc7ddc77d7ab4aaf77a0a3e85c649b496